src/ray/common/ray_config_def.h

```diff
@@ -198,3 +198,10 @@ RAY_CONFIG(uint32_t, object_store_get_warn_per_num_attempts, 50)
 /// When getting objects from object store, max number of ids to print in the warning
 /// message.
 RAY_CONFIG(uint32_t, object_store_get_max_ids_to_print_in_warning, 20)
+
+/// Allow up to 5 seconds for connecting to GCS Service.
```

Comment:
add a comment that this only takes effect when gcs service is enabled.

Reply:
fixed